product_docs/docs/biganimal/release/administering_cluster/01_portal_access.mdx

```diff
@@ -91,3 +91,34 @@ Organization owners can assign users organization-level roles to complete certai
 5.  Select **Submit**.
 
 See [Adding a user to a project](/biganimal/release/administering_cluster/projects.mdx) for information on adding users to projects.
+
+## Machine users
+
+A machine user is used to drive non-GUI access to BigAnimal by user customized automation script, BigAnimal CLI commands, Terraform manifest, BigAnimal's first party binaries that runs from your environment.
+
+- Any user with the organization's owner role can create, update and delete a machine user.
```

Comment:
should it be `Owner role` or `owner role`? do we usually capitalized user roles in the BA docs or keep them lowercase?

Reply:
it's owner role, lowercase across the docs